tests/unit/test.program.js

```diff
@@ -417,6 +417,17 @@ describe('program.main', () => {
         assert.equal(customPrefs.prop2, 'value2');
       });
   });
+
+  it('pass shouldExitProgram option to commands', () => {
+    const fakeCommands = fake(commands, {
+      lint: () => Promise.resolve(),
+    });
+    return execProgram(['lint'], {commands: fakeCommands}
+    ).then(() => {
```

Comment:
style nit: I think this would be easier to read if it moved up a line like this:

````js
return execProgram(['lint'], {commands: fakeCommands}).then(() => {
````
If that exceeds 80 chars, try to break it more like this:
````js
return execProgram(
  ['lint'], {commands: fakeCommands}).then(() => {
````